docs/changelogs/CHANGELOG-1.3.md

```diff
@@ -43,6 +44,7 @@
   - Upgrade Flannel to v0.14.0
   - Upgrade Kubernetes dashboard to v2.3.1
   - Upgrade Kubernetes metrics-scraper to v1.0.7
+- [#2747](https://github.com/epiphany-platform/epiphany/issues/2747) - Upgrade Node-Exporter to v1.3.1
```

Comment:
Could you place 2 new lines close to each other as they are related to similar topic?